wolfssl/openssl/evp.h

```diff
@@ -247,6 +247,7 @@ enum {
     NID_rc4           = 5,
     EVP_PKEY_DH       = NID_dhKeyAgreement,
     EVP_PKEY_HMAC     = NID_hmac,
+    EVP_PKEY_FALCON = 300,
```

Comment:
How'd you come up with 300?

Reply:
Pull it out of thin air.  Do you have a suggestion for something different?

Reply:
No that is fine, but please comment as such. Usually these are NID values.

Reply:
Got it...will add a comment.